microhapulator/data/marker_details_template.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" media="screen">
+
+    <title>MicroHapulator Marker Detail Report</title>
+    <style type="text/css">
+        .container {
+            max-width: 750px;
+            padding-top: 100px;
+        }
+
+        .alnrt {
+            text-align: right;
+        }
+
+        table {
+            margin: 12pt auto;
+            width: 100%;
+        }
+
+        td,
+        th {
+            padding: auto 5px;
+            white-space: nowrap;
+        }
+
+        .offsets {
+            overflow-wrap: break-word;
+        }
+
+        .sequence {
+            border: none;
+            padding: 5px;
+            width: 100%;
+            height: 40px;
+            overflow-x: scroll;
+            font-family: 'Courier New', monospace;
+            font-size: 12pt;
+        }
+
+        ::-webkit-scrollbar {
+            width: 5px;
+            height: 5px;
+        }
+
+        ::-webkit-scrollbar-thumb {
+            background: gray;
+            border-radius: 10px;
+        }
+
+        ::-webkit-scrollbar-track {
+            -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+            border-radius: 10px;
+            background-color: #F5F5F5;
+        }
+    </style>
+</head>
+
+<body>
+
+
+    <div class="container" id="marker_detail_container">
+        <select onchange="show_selected_details(this.value)" id="drop_down">
+            {% for markername in markernames %}
+            <option>{{markername}}</option>
+            {% endfor %}
+        </select>
+        <input id="searchbar" , type="search" onsearch="search_marker()" name="search" placeholder="Search markers..">
+        <p></p>
+        {% for markername in markernames %}
+        <div class="marker_details" id={{markername}} style="display:none">
+            <p>
+            <h4>Length</h4>{{marker_details_table.loc[markername,'Length']}} bp</p><br>
+            <p>
```

Comment:
Typically, `<h?>` tags aren't nested within `<p>` tags, which are used for the text of a paragraph's body. The following would be more conventional.

```html
<h4>Length</h4>
<p>{{marker_details_table.loc[markername,'Length']}} bp</p>
<br />
```